legacy/application/forms/AddShowAutoPlaylist.php

```diff
@@ -34,6 +34,36 @@ public function init()
             'class' => 'input_text',
             'decorators' => ['ViewHelper'],
         ]);
+
+        // Add override intro playlist checkbox element
+        $this->addElement('checkbox', 'add_show_override_intro_playlist', [
+            'label' => _('Override Intro Playlist ?'),
+            'required' => false,
+            'class' => 'input_text',
+            'decorators' => ['ViewHelper'],
+        ]);
+
+        $introPlaylistSelect = new Zend_Form_Element_Select('add_show_intro_playlist_id');
+        $introPlaylistSelect->setLabel(_('Select Intro Playlist'));
+        $introPlaylistSelect->setMultiOptions(Application_Model_Library::getPlaylistNames(true));
```

Comment:
I don't know how expensive `Application_Model_Library::getPlaylistNames(true)` is, but maybe we can store the result in a variable and not call the DB twice?